tests/activity_sticky_queues/activity_sticky_queues_activity_test.py

```diff
@@ -0,0 +1,35 @@
+from pathlib import Path
+from unittest import mock
+
+from activity_sticky_queues import tasks
+
+RETURNED_PATH = "valid/path"
+tasks._get_delay_secs = mock.MagicMock(return_value=0.0001)
+tasks._get_local_path = mock.MagicMock(return_value=Path(RETURNED_PATH))
+
+
+async def test_download_activity():
+    worker_id = "an-id"
+    workflow_uuid = "uuid"
+    want = "/".join([RETURNED_PATH, worker_id, workflow_uuid])
```

Comment:
This is failing on Windows due to dir separator differences :-) Can use `pathlib` or something

Reply:
Done

Reply:
Done